staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -5411,3 +5480,37 @@ func loadBalancerStatusStringer(s corev1.LoadBalancerStatus, wide bool) string {
 	}
 	return r
 }
+
+// searchEvents finds events about the specified object.
+// It is very similar to CoreV1.Events.Search, but supports the Limit parameter.
```

Comment:
Here's the Search implementation for reference: https://github.com/kubernetes/kubernetes/blob/6aa683e9cf9bfa9069a0085eb8a1756fc0678e07/staging/src/k8s.io/client-go/kubernetes/typed/core/v1/event_expansion.go#L100-L123

Two other meaningful differences to flag for the purposes of review:
- Don't need to pass in `scheme.Scheme` as an argument because it is all local
- Can use the namespace in the ref directly instead of having `client.Events` use a separate ns arg, since it is built internally. I think this actually fixes a bug in the PV and StorageClass describers. Neither of these is namespaced, but previously they passed in the user-provided ns instead of the empty "all" namespace constant. Tests didn't catch it because the fake for the event client's `Search` is super different from the real thing in terms of namespace handling.